src/gui/src/layoutViewer.cpp

```diff
@@ -2304,6 +2310,9 @@ void LayoutViewer::paintEvent(QPaintEvent* event)
     painter.setBrush(QBrush());
     painter.drawRect(rubber_band_.normalized());
   }
+
+  // painting is done, okay to update outputs agai
+  output_widget_->bufferOutputs(false);
```

Comment:
What triggers the buffered output to actually be printed?  Does it wait for the next event?

Reply:
The outputs are queued as events by QT, so when the paintEvent is done, the next set of events should update the ScriptWidget to reflect the text.

Reply:
There could be no next event queued.  Since this is mostly for debug I guess it isn't too important a corner case to worry over.